packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -137,7 +166,7 @@ class _Elements {
       paint: parserState.parseStyle(parserState.rootBounds, parent.paint,
           currentColor: color),
       transform:
-          parseTransform(parserState.attribute('transform'), parent.transform),
+          parseTransform(parserState.attribute('transform')),
```

Comment:
I've made all the transforms "lazy" - they are now applied while building the nodes themselves. This made it significantly easier to fix a bug where nodes create by `use` had a different transform then was indended.